0-python-basics/calculate/calculate.py

```diff
@@ -4,15 +4,22 @@
 def calculate(e: str) -> float:
     g = []
     nums = re.findall('[0-9]+', e)
+
     for i in nums:
         g.append(float(i))
+
     a = float(g[0])
+
```

Comment:
Use blank lines before and after the `if`, `for`, `while`, `return` operators. No need to have a blank line here because these operations are closely related.